onboarding-manager/internal/invclient/invclient.go

```diff
@@ -407,6 +407,12 @@ func (c *OnboardingInventoryClient) UpdateHostStateAndRuntimeStatus(ctx context.
 		computev1.HostResourceFieldHostStatus,
 		computev1.HostResourceFieldHostStatusIndicator,
 		computev1.HostResourceFieldHostStatusTimestamp,
+		computev1.HostResourceFieldOnboardingStatus,
```

Comment:
`UpdateHostStateAndRuntimeStatus` is this a shared function? We risk to reset some fields

Reply:
It is only called from the `invalidateHost` path in the host reconciler